shell/platform/windows/compositor_software.h

```diff
@@ -15,7 +15,8 @@ namespace flutter {
 // rasterization and bitmaps.
 class CompositorSoftware : public Compositor {
  public:
-  CompositorSoftware(FlutterWindowsEngine* engine);
+  CompositorSoftware(PlatformViewManager* manager,
+                     FlutterWindowsEngine* engine);
```

Comment:
Super silly nitpick: I usually order arguments by "importance of types". In this case, I'd consider the engine to be more important and would place it before the platform view manager.

Reply:
You also suggested we wait to add the manager to the constructor for now anyway, correct?

Reply:
Yup that's right. I think I left this comment before I realized this wasn't used yet and forgot to remove this.